ote/src/clj/ote/tasks/pre_notices.sql

```diff
@@ -37,8 +39,11 @@ SELECT to_char(chg."change-date", 'dd.mm.yyyy') as "change-date",
        ts.id AS "transport-service-id"
   FROM changes_with_regions chg
   JOIN "transport-service" ts ON ts.id = chg."transport-service-id"
-  JOIN "transport-operator" op ON op.id = ts."transport-operator-id"
+  JOIN "transport-operator" op ON op.id = ts."transport-operator-id",
+       gtfs_package p
  WHERE chg.date = CURRENT_DATE
+    AND p.id = ANY(chg."package-ids")
+   AND p.created > CURRENT_DATE - interval '8 hours'
```

Comment:
Will this assume package creation done at fixed time 08.15 and break if that changes? 
If so, instead of shifting 8h could this use CURRENT_DATE anbd time of hour 00.00 or 00.01?

Reply:
I'm not sure, but I think that we have a mismatch in datetime values in db. Some of them are created using now() function in db and some of them using java.datetime. And those differences might cause mismatches  when comparing dates in database.

Our automatic change detection is timed to start at 05.15. And gtfs import is timed to start at 00.00,  But due to my assumption that times are in different timezones I add 8 hour window to skip any timezone differences (3 hours at summer, 2 hours at winter time)